cmake/modules/SwiftSystemConfig.cmake.in

```diff
@@ -0,0 +1,12 @@
+#[[
+This source file is part of the Swift System open source Project
+
+Copyright (c) 2021 Apple Inc. and the Swift System project authors
+Licensed under Apache License v2.0 with Runtime Library Exception
+
+See https://swift.org/LICENSE.txt for license information
+#]]
+
+if(NOT TARGET SwiftSystem)
```

Comment:
This needs to change to `SystemPackage` since you renamed the target.

Reply:
```suggestion
if(NOT TARGET SystemPackage)
```